internal/service/amplify/branch.go

```diff
@@ -126,7 +126,6 @@ func ResourceBranch() *schema.Resource {
 			"enable_performance_mode": {
 				Type:     schema.TypeBool,
 				Optional: true,
-				ForceNew: true,
```

Comment:
Are the underlying API calls to AWS verified to work with this change?

Reply:
yes, it seems to be ok.
```console
$ terraform plan
  # aws_amplify_branch.master will be updated in-place
  ~ resource "aws_amplify_branch" "master" {
      ~ enable_performance_mode     = false -> true
        id                          = "d2rpmmwfwa1zzl/master"
        tags                        = {}
        # (14 unchanged attributes hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```